package.json

```diff
@@ -15,17 +16,17 @@
     "dev": "concurrently -p none \"yarn start:main\" \"yarn start:renderer\" ",
     "dev:android": "yarn build:renderer && concurrently -p none \"yarn start:renderer\" \"yarn cap run android\" ",
     "build": "concurrently -p none \"yarn build:main\" \"yarn build:renderer\" \"yarn gen:all\" && yarn cap sync",
-    "build:main": "concurrently -p none \"tsc ./src/main/preload.ts\" \"tsc -p ./src/main/tsconfig.json\" ",
+    "build:main": "tsc -p ./src/main/tsconfig.json",
```

Comment:
not running the preload.ts might cause problems if the preload.js doesn't exist, try to delete preload.js locally and see if it will transpile it

Reply:
running `tsc -p ./src/main/tsconfig.json` causes all files to be transpiled, including preload.ts into `release/dist` which we then load the main.js from as you can see:

<img width="1037" alt="Screenshot 2025-05-10 at 10 44 14" src="https://github.com/user-attachments/assets/4f13fa90-e399-4961-a05f-3e624758ab16" />

it was previously loaded from `./src/main/main.ts` but this is no longer the case.

removing the `release` folder and then re-running `yarn dev` works just fine.